src/framework/components/rigid-body/component.js

```diff
@@ -117,24 +113,20 @@ pc.extend(pc, function () {
                 if (this.body) {
                     var vel = this.body.getAngularVelocity();
                     this._angularVelocity.set(vel.x(), vel.y(), vel.z());
-                    return this._angularVelocity;
                 }
-            } else {
-                return this._angularVelocity;
             }
+            return this._angularVelocity;
```

Comment:
Just wanna raise a flag that previously if `this.body` was falsy, no value would be returned (ie: `undefined` would be returned). I assume the new code is *more* correct, but since this PR is not really about changing code behavior, I just wanted to point that out. Is this an intentional change?

Reply:
'''this.body''' is only null if Ammo can't be loaded for whatever reason. Angular and linear velocity for a rigidbody component should be 0,0,0 rather than undefined in this case. The fact that there was a code path in those getters without a return was a bug. This PR is about changing code behavior if that behavior is wrong. 😄

Reply:
Great :)